packages/documentation-site/patternfly-docs/content/design-guidelines/styles/colors/colors.md

```diff
@@ -174,4 +188,43 @@ Status and state colors are indicators that communicate data and actions to user
 
 [Our goal](/accessibility/patternflys-accessibility/) is for PatternFly to meet [level AA requirements in the Web Content Accessibility Guidelines 2.1](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-21/). To achieve level AA accessibility, your UI contrast ratios must be at or above 4.5:1 for normal text, 3:1 for large text, and 3:1 for graphics and other UI components. Additionally, on hover, link text color should have ample contrast from both the background color and the default state link color.
 
-To check the contrast between background and text colors, use a <a href="https://color.a11y.com/?wc3" target="_blank" className="pf-m-link">WCAG AA-compliance tool.</a>
\ No newline at end of file
+To check the contrast between background and text colors, use a <a href="https://color.a11y.com/?wc3" target="_blank" className="pf-m-link">WCAG AA-compliance tool.</a>
+
+<Divider/>
+
+## PatternFly palettes
+Our color palettes are created as a system, designed to reinforce content and workflows. Our palettes are carefully chosen to support effective communication and adapt to the needs of your UI.
+
+Note that bright colors are used sparingly and are reserved for specific interactions, such as communicating status.
+
+Our color palettes are organized into "families" that contain different shades of the same hue. In the following palettes, you can expand each value color or shadow in a family to see more details.
```

Comment:
```suggestion
## Color families 
Our color palettes are organized into "families" that contain different shades of the same hue. In the following families, you can expand each color to see related tokens.
```

Did we decide to keep the color families in addition to linking out to the Red Hat palettes? I think I removed the whole section in my pr, so would update the text to something like this if we are going to keep them